frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import { Accordion } from '@digdir/design-system-react';
 import { FileIcon } from '@navikt/aksel-icons';
```

Comment:
I know this is not from this PR, but can we change the import of this icon so that we "clean up" on the go? 😄

Reply:
Yeah, nice catch :)

Reply:
This is not related to this issue, but I hope it is ok:
I used the search function in VSC to replace all of those navikt/aksel-icon imports (except where its converted to @studio-icons)